Zappy_IA_Report/DTC/Code/game.py

```
@@ -0,0 +1,45 @@
+def remove_client(dead_client) -> None :
+    global clients
+
+    clients.remove(dead_client)
+
+def cleanup_thread() -> None :
+    global threads
+
+    alive_thread = [t for t in threads if t.is_alive()]
+    threads = alive_thread
+
+def game(process_args) -> None :
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)